lib/src/widgets/animated_button.dart

```diff
@@ -130,6 +126,40 @@ class _AnimatedButtonState extends State<AnimatedButton>
     }
   }
 
+  // initializes width and size animation
+  void _initWidth(BuildContext context) {
```

Comment:
Change the method name to `_updateWidth` and add the following code in `didUpdateWidget` so everytime the label change (mostly for debugging) the button's width will be updated too

```
    if (oldWidget.text != widget.text) {
      _updateWidth(context);
    }
```